insights/specs/__init__.py

```diff
@@ -491,6 +491,7 @@ class Specs(SpecSet):
     uptime = RegistryPoint()
     usr_journald_conf_d = RegistryPoint(multi_output=True)
     vdsm_conf = RegistryPoint()
+    tuned_conf = RegistryPoint()
```

Comment:
Better insert this spec as per the alphabet order.

Reply:
Done